src/java/org/apache/cassandra/db/ReadCommand.java

```diff
@@ -595,8 +595,8 @@ public WithoutPurgeableTombstones()
     public String toCQLString()
     {
         StringBuilder sb = new StringBuilder();
-        sb.append("SELECT ").append(columnFilter());
-        sb.append(" FROM ").append(metadata().ksName).append('.').append(metadata.cfName);
+        sb.append("SELECT ").append(columnFilter().toCQLString());
+        sb.append(" FROM ").append(metadata().ksName).append('.').append(metadata().cfName);
```

Comment:
We do not quote keyspace and table names if needed.

I am not sure how much we want to fix things in this patch.